projects/1.20/assets/gobber/gobber2/lang/zh_cn.json

```
@@ -0,0 +1,580 @@
+	"item.gobber2.gobber2_armor.line2b": "头盔：水下呼吸",
+	"item.gobber2.gobber2_armor.line3": "胸甲：无",
+	"item.gobber2.gobber2_armor.line4": "护腿：免疫摔落伤害",
+	"item.gobber2.gobber2_armor.line5": "靴子：无",
+	"item.gobber2.gobber2_armor.line6": "全套加成：免疫中毒",
+	"item.gobber2.gobber2_helmet": "戈伯头盔",
+	"item.gobber2.gobber2_chestplate": "戈伯胸甲",
+	"item.gobber2.gobber2_leggings": "戈伯护腿",
+	"item.gobber2.gobber2_boots": "戈伯靴子",
+
+	"item.gobber2.gobber2_armor_nether.line2": "头盔：中速自动补食、饱和、水下呼吸和伤害吸收",
+	"item.gobber2.gobber2_armor_nether.line2b": "头盔：水下呼吸和伤害吸收",
+	"item.gobber2.gobber2_armor_nether.line3": "胸甲：抗火和击退抗性",
+	"item.gobber2.gobber2_armor_nether.line4": "护腿：免疫摔落伤害",
+	"item.gobber2.gobber2_armor_nether.line5": "靴子：无",
+	"item.gobber2.gobber2_armor_nether.line6": "全套加成：免疫中毒和凋零",
+	"item.gobber2.gobber2_helmet_nether": "下界头盔",
+	"item.gobber2.gobber2_chestplate_nether": "下界胸甲",
+	"item.gobber2.gobber2_leggings_nether": "下界护腿",
+	"item.gobber2.gobber2_boots_nether": "下界靴子",
+
+	"item.gobber2.gobber2_armor_end.line1": "头盔：快速自动补食、饱和、水下呼吸和伤害吸收",
+	"item.gobber2.gobber2_armor_end.line1b": "头盔：水下呼吸和伤害吸收",
+	"item.gobber2.gobber2_armor_end.line2": "胸甲：抗火和击退抗性",
+	"item.gobber2.gobber2_armor_end.line3": "护腿：免疫摔落伤害",
+	"item.gobber2.gobber2_armor_end.line4": "靴子：无",
+	"item.gobber2.gobber2_armor_end.line5": "全套加成：免疫负面效果",
+	"item.gobber2.gobber2_helmet_end": "末地头盔",
+	"item.gobber2.gobber2_chestplate_end": "末地胸甲",
+	"item.gobber2.gobber2_leggings_end": "末地护腿",
+	"item.gobber2.gobber2_boots_end": "末地靴子",
+
+	"item.gobber2.gobber2_armor_dragon.line1": "头盔：快速自动补食、饱和、水下呼吸和伤害吸收",
+	"item.gobber2.gobber2_armor_dragon.line1b": "头盔：水下呼吸和伤害吸收",
+	"item.gobber2.gobber2_armor_dragon.line2": "胸甲：抗火和击退抗性",
+	"item.gobber2.gobber2_armor_dragon.line3": "护腿：免疫摔落伤害",
+	"item.gobber2.gobber2_armor_dragon.line4": "靴子：无",
+	"item.gobber2.gobber2_armor_dragon.line5": "全套加成：创造模式飞行、免疫负面效果",
+	"item.gobber2.gobber2_armor_dragon.line6": "全套加成：免疫负面效果",
+	"item.gobber2.gobber2_helmet_dragon": "龙之头盔",
+	"item.gobber2.gobber2_chestplate_dragon": "龙之胸甲",
+	"item.gobber2.gobber2_leggings_dragon": "龙之护腿",
+	"item.gobber2.gobber2_boots_dragon": "龙之靴子",
+
+	"item.gobber2.gobber2_links_end": "末地盔甲锁链",
+
+	"item.gobber2.sneak_right_click": "潜行右击切换开/关",
+	"item.gobber2.works_in_enderchest": "在末影箱中生效",
+	"item.gobber2.works_in_inventory": "在物品栏中生效",
+	"item.gobber2.works_in_main_hand": "在主手时生效",
+	"item.gobber2.works_in_off_hand": "在副手时生效",
+	"item.gobber2.right_click_use": "右击使用",
+
+	"item.gobber2.gobber2_ring.cooldown": "冷却时间（刻）：%s",
+	"item.gobber2.gobber2_ring": "戒指",
+	"item.gobber2.gobber2_ring_nether": "下界戒指",
+	"item.gobber2.gobber2_ring_end": "末地戒指",
+
+	"item.gobber2.gobber2_ring_above": "乾坤戒指",
+	"item.gobber2.gobber2_ring_above.line1": "将玩家传送到正上方最高处的安全方块上",
+	"item.gobber2.gobber2_ring_above.line2": "右击使用",
+	"item.gobber2.gobber2_ring_above.line3": "潜行右击使用时将玩家传送到正下方最低处的安全方块上",
+	"item.gobber2.gobber2_ring_above.line4": "请注意，向下传送时，您可能会被传送到非常底层的位置！",
```

```suggestion
	"item.gobber2.gobber2_ring_above.line4": "请注意，向下传送时，可能会被传送到非常低的位置！",
```
底层……有点怪，不过也可以保留
总之我们翻译叙述性文本的时候不用敬语，一般英语里的第二人称也会省略